datafusion/sqllogictest/test_files/expr.slt

```diff
@@ -1472,6 +1472,135 @@ SELECT extract(epoch from arrow_cast('1969-12-31', 'Date64'))
 ----
 -86400
 
+# test_extract_interval
```

Comment:
Can you please add some tests for:

1. Tables with intervals (including some null values) rather than just scalars
2. Nulls
?